### PR TITLE
* added dark mode images when using iconset-pdf option.

### PR DESCRIPTION
**Pull request by owenzhao**
I didn't test the other options, however, at least they should work as before.

### CODE REVIEW

File: Sources/SFSymbolsCore/Exporters/IconsetExporter.swift [hunk 1]
```diff
@@ -28,30 +28,32 @@ public struct IconsetExporter: Exporter {
         
         for glyph in font.glyphs(matching: options.matchPattern) {
             try autoreleasepool {
-                try exportGlyph(glyph, in: font, to: assetFolder)
```
Comment (kireev165) [inline Sources/SFSymbolsCore/Exporters/IconsetExporter.swift]: Why did you remove `autoreleasepool`?

File: Sources/SFSymbolsCore/Exporters/IconsetExporter.swift [hunk 2]
```diff
@@ -95,43 +96,61 @@ public struct PDFAssetCatalog: Exporter {
         try Data(contentsJSON.utf8).write(to: contentsURL)
         
         for glyph in font.glyphs(matching: options.matchPattern) {
-            try autoreleasepool {
```
Comment (arvaki) [inline Sources/SFSymbolsCore/Exporters/IconsetExporter.swift:98]: Why did you remove `autoreleasepool`?

**Comment by arvaki** [comment]
I am against this change. IMO this repo has nothing to do with light/dark schemes.

**Comment by davedelong** [comment]
I think the better change would be to specify that the PDFs in the iconset are templates. Then they'll inherit the color scheme of the app's environment.

**Comment by owenzhao** [comment]
I think there are two issues here. 

1. You are using iconset, but for using as image assets in Xcode, it should be imageset. Iconset is used only for icon, which is shown in mac as image icon. But this project's propose is to use those images as toolbar or menu images. So all iconset part should change to imageset. 

2. I have already changed iconset to imageset in my ported project. Both pdf and png imagesetters will work properly as toolbar or menu images.
